src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Test is invalid. It's gonna fail, either remove tests or fix them according to your code.